crates/wash-lib/src/parser/mod.rs

```diff
@@ -1091,6 +1154,32 @@ impl ProjectConfig {
     }
 }
 
+/// Helper function to canonicalize a path or create it if it doesn't exist before
+/// attempting to canonicalize it. This is a nice helper to ensure that we can attempt
+/// to precreate directories like `build` before we start writing to them.
+fn canonicalize_or_create(path: PathBuf) -> Result<PathBuf> {
```

Comment:
_Technically_ we should probably be using the tokio one, but since this is a CLI we can probably get away with it

Reply:
Oh true, I do call this from an async context so let me use that